src/third_party/html-parser/html-parser.ts

```diff
@@ -232,3 +232,15 @@ export function parse(data: string, rootElement: Element) {
 
   throw new Error('Attempting to parse invalid HTML.');
 }
+
+function decodeNumericEntities(html: string) {
+  return html.replace(/&#(x?\d+);?/g, function (s, entity) {
```

Comment:
We need to parse hex entities and number entities.
```suggestion
  return html.replace(/&#(?:x([a-f\d]+)|(\d+));?/ig, function (s, hex, num) {
```

Reply:
ah right, hex means letters too